CHANGELOG.md

```diff
@@ -1,9 +1,22 @@
 ## 6.0.0 (YYYY-MM-DD)
 
+The feature previously named Partial Sync is now called Query-Based Sync and is now the default mode when synchronizing Realms.
+This has impacted a number of API's. See below for the details.
+
 ### Breaking Changes
 
-* [ObjectServer] Partial sync is now the default mode of synchronization. `SyncConfiguration.Builder.partialRealm()` has been replaced by `SyncConfiguration.Builder.fullSynchronization()`.
-* [ObjectServer] `SyncConfiguration.isPartialRealm()` has been replaced by `SyncConfiguration.isFullySynchronizedRealm()`.
+* [ObjectServer] `SyncConfiguration.automatic()` has been replaced by `SyncUser.getDefaultConfiguration()`.
```

Comment:
Why not just deprecated so we avoid making a breaking change and a major version update?
Related: This should no goto next-major branch or be 6.0 then.

Reply:
Yes, that was the plan. Looks like an oversigt when reverting the breaking changes.